staging/src/k8s.io/client-go/transport/token_source_test.go

```diff
@@ -140,16 +140,23 @@ func TestCachingTokenSourceRace(t *testing.T) {
 
 		var wg sync.WaitGroup
 		wg.Add(100)
+		errc := make(chan error)
```

Comment:
to avoid the possibility of a hang, the channel must be able to hold as many errors as we spawn goroutines

Reply:
@liggitt Thanks for your review! Please check again.